src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java

```diff
@@ -481,21 +481,25 @@ private void checkFilesForConsistencyRegardingTheirKeys(Map<File, Set<String>> f
             translations.load(inStream);
             keys = translations.stringPropertyNames();
         }
-        catch (final IOException ex) {
-            logIoException(ex, file);
+        catch (final IOException | IllegalArgumentException ex) {
+            logException(ex, file);
```

Comment:
@romani The reason why I choose IAE is because `Exception` is disallowed by our own configuration.
> TranslationCheck.java:483:9: Catching 'Exception' is not allowed. [IllegalCatch]

Do you still want to continue with catching Exception and suppress violation?
If so, is it better to just disable this in `IllegalCatch` in our config?

Reply:
>Do you still want to continue with catching Exception and suppress violation?

Catching all exceptions are highly discouraged but unavoidable in real life, so should be very very limited.
Can we catch all in Checker and continue with next file? I think we already for this, we probably need to expand scope of that catch block.

Reply:
> Can we catch all in Checker and continue with next file? 

Current catch in checker is during normal processing. Exception in Translation is happening during `finishProcessing` where there isn't any catch block at all since it is doing multi-file check work. Even if there was a catch block, it wouldn't tell us the file being processed since Translation is doing its own files as a multi-file check.

Reply:
Yep. In this case such Check should catch all in such methods, it will not support mode of Checker to continue on exception, but it is ok for now

Reply:
@romani So I assume you want me to still catch Exception and suppressing violation?

Reply:
Yes, we do this on good reason.
Violation just try to prevent us from accidental mistake. 

Eventually, we might use single xpath filter here, as all multifile Checks catch all in such methods, or ....

Reply:
Done.